tests/test_fiod.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -xeo pipefail
+
+source tests/common.sh
+
+function finish {
+  echo "Cleaning up Fio"
+  kubectl delete -f tests/test_crs/valid_fiod.yaml
+  delete_operator
+}
+
+trap finish EXIT
+
+function functional_test_fio {
+  apply_operator
+  kubectl apply -f tests/test_crs/valid_fiod.yaml
+  check_pods 3
+  fio_pod=$(kubectl get pods -l app=fio-client --namespace ripsaw -o name | cut -d/ -f2)
+  kubectl wait --for=condition=Initialized "pods/$fio_pod" --namespace ripsaw --timeout=200s
+  kubectl wait --for=condition=complete -l app=fio-client jobs --namespace ripsaw --timeout=100s
```

Comment:
has to be be `app=fiod-client` as the label has been updated in the client definition